ganga/GangaGUI/config.py

```diff
@@ -0,0 +1,7 @@
+"""Contains the configuration settings for the GUI Flask App"""
+
+import os
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get("SECRET_KEY") or "f3aa26t8b537abf6ee6305eefea0a10a"
```

Comment:
I guess this is just something temporary ...

Reply:
Yes, but the user will always need to specify a secret key in order to run Flask. We can maybe generate it in `.gangarc`, and then take it from there in the future?